@here/harp-datasource-protocol/lib/TechniqueParams.ts

```diff
@@ -880,7 +884,7 @@ export interface ExtrudedPolygonTechniqueParams extends StandardTechniqueParams
     /**
      * Animate the extrusion of the buildings if set to `true`.
      */
-    animateExtrusion?: MaybeInterpolatedProperty<boolean>;
```

Comment:
We actually have interpolated animatedExtrusion in our styles here:
https://github.com/heremaps/harp.gl/blob/master/%40here/harp-map-theme/resources/berlin_tilezen_base.json#L1238

Reply:
Sorry, reverted this during development. Reverting change.